urwid/__init__.py

```diff
@@ -242,3 +244,46 @@
 
 # Backward compatibility
 VERSION = __version_tuple__
+
+
+# Moved modules handling
+__locals: dict[str, typing.Any] = locals()  # use mutable access for pure lazy loading
+
+# Backward compatible lazy load with deprecation warnings
+_moved_warn: dict[str, str] = {
+    "escape": "urwid.display.escape",
+    "lcd_display": "urwid.display.lcd",
+    "html_fragment": "urwid.display.html_fragment",
+    "web_display": "urwid.display.web",
+}
+# Backward compatible lazy load without any warnings
+# Before DeprecationWarning need to start PendingDeprecationWarning process.
+_moved_no_warn: dict[str, str] = {
```

Comment:
IMHO, no any warnings/deprecation should be before the next API breaking release (when `DeprecationWarning` will be dropped and `PendingDeprecationWarning` became `DeprecationWarning`).
Then should be long cycle of deprecation (`PendingDeprecationWarning` -> `DeprecationWarning`).
This is too wide used API